### ISSUE TITLE
Android Basics: Lunch Tray app your test cases wrong at NavigationTests.kt

### ISSUE THREAD

**Issue by utkuoruc** [vishnurajanme/3-Kotlin-Codelab-Lunch-Tray#1]
hello there;
I was checking your repository to solve a little problem I had, then I decided to run tests in your project. In NavigationTests.kt file,
change R.id.layout_start_order to R.id.startOrderFragment
that's why your tests are failing

have a nice one

**Comment by vishnurajanme** [comment]
Yup. I hadnt changed the tests part as they mentioned in the codelab not
to. Thank you for spotting the error
